src/main/java/edu/utdallas/davisbase/representation/InsertCommandRepresentation.java

```diff
@@ -0,0 +1,61 @@
+package edu.utdallas.davisbase.representation;
+
+import java.util.List;
+
+public class InsertCommandRepresentation implements CommandRepresentation {
+  private String command;
+  private String table;
+  private List<String> columns;
+  private List<String> values;
+
+  public InsertCommandRepresentation(String command, String table, List<String> columns, List<String> values) {
+    this.command= command;
+    this.table = table;
+    this.columns = columns;
+    this.values = values;
+  }
+
+  public String getTable() {
+    return table;
+  }
+
+  public void setTable(String table) {
+    this.table = table;
+  }
+
+  public List<String> getColumns() {
+    return columns;
+  }
+
+  public void setColumns(List<String> columns) {
```

Comment:
Remove the setters to make this object immutable. _(it provides stronger compile-time correctness guarantees, and we don't the setters after construction anyway)_